contracts/IdentityRegistryBridger.sol

```diff
@@ -108,6 +142,45 @@ contract IdentityRegistryBridger is ERC721TokenReceiver {
         serviceRegistry = _serviceRegistry;
     }
 
+    /// @dev Registers 8004 agent Id corresponding to service Id.
+    /// @param serviceId Service Id.
+    /// @param multisig Service multisig.
+    /// @param tokenUri Service tokenUri.
+    /// @return agentId Corresponding 8004 agent Id.
+    function _register(uint256 serviceId, address multisig, string memory tokenUri) internal returns (uint256 agentId)  {
```

Comment:
The internal of this stripped out function has not changed from the previous PR.